scripts/install/mylar.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+# Mylar installer
+# Author: Brett
+# Copyright (C) 2021 Swizzin
+# Licensed under GNU General Public License v3.0 GPL-3 (in short)
+#
+#   You may copy, distribute and modify the software as long as you track
+#   changes/dates in source files. Any modifications to our software
+#   including (via compiler) GPL-licensed code must also be made available
+#   under the GPL along with build & install instructions.
+#
+#! /bin/bash
+#shellcheck source=sources/functions/utils
+. /etc/swizzin/sources/functions/utils
+#shellcheck source=sources/functions/pyenv
+. /etc/swizzin/sources/functions/pyenv
+
+user=$(_get_master_username)
+port=$(port 7000 11000)
```

Comment:
not pinning the port?

Reply:
I guess we could since it's only a single-user app.